<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggerExampleDialog</class>
 <widget class="QDialog" name="LoggerExampleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoggerExampleDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>日志行数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="recordNums"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="loggingToFile">
      <property name="text">
       <string>直接记录到文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="testPolicy">
      <property name="text">
       <string>使用Policy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="testMoonLogger">
      <property name="text">
       <string>使用MoonLogger</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
